src/CoreGraphics/CGColorSpace.cs

```diff
@@ -378,6 +379,50 @@ public NSData GetICCProfile ()
 			IntPtr ptr = CGColorSpaceCopyICCProfile (handle);
 			return (ptr == IntPtr.Zero) ? null : new NSData (ptr, true);
 		}
+
+		[iOS (10,0)][Mac (10,12)]
+		[DllImport (Constants.CoreGraphicsLibrary)]
+		static extern /* CFDataRef* */ IntPtr CGColorSpaceCopyICCData (/* CGColorSpaceRef */ IntPtr space);
+
+		[iOS (10,0)][Mac (10,12)]
+		public NSData GetIccData ()
+		{
+			IntPtr ptr = CGColorSpaceCopyICCData (handle);
+			return (ptr == IntPtr.Zero) ? null : new NSData (ptr, true);
+		}
+
+		[iOS (10,0)]
+		[DllImport (Constants.CoreGraphicsLibrary)]
+		static extern unsafe /* CFStringRef* */ IntPtr CGColorSpaceCopyName (/* CGColorSpaceRef */ IntPtr space);
+
+		[iOS (10,0)]
+		public string Name {
+			get {
+				return CFString.FetchString (CGColorSpaceCopyName (handle));
```

Comment:
It returns `null`, which makes it property return `null` - but I should add a test in case it change

In general (at least in generated code, it's not very consistent in manual bindings) we do not check all handle access as it would add a lot of code to the assemblies.